plotly_resampler/figure_resampler/utils.py

```diff
@@ -128,7 +128,7 @@ def timedelta_to_str(td: pd.Timedelta) -> str:
     if c.days > 0:
         out_str += f"{c.days}D"
     if c.hours > 0 or c.minutes > 0 or c.seconds > 0 or c.milliseconds > 0:
-        out_str += "_" if len(out_str) else ""
+        out_str += "_" if out_str else out_str
```

Comment:
Again, this is here somewhat less clear for me (might be personal flavor).
e.g., I would at lease propose to retain the  `else ""`  part.

Reply:
I see your point there -- some of the refurb suggestions I also disagree with! Will make this change